pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -34,20 +34,21 @@ bool IsTorchMlirSupported(const torch::jit::Node& node) {
 
 std::unordered_set<std::string> GetTorchMlirWhiteList() {
   return std::unordered_set<std::string>{
-      "aten::relu",     "aten::relu6",       "aten::leaky_relu",
-      "aten::sigmoid",  "aten::glu",         "aten::contiguous",
-      "aten::erf",      "aten::exp",         "aten::neg",
-      "aten::rsqrt",    "aten::neg",         "aten::to.dtype",
-      "aten::type_as",  "aten::bitwise_not", "aten::to",
-      "aten::gelu",     "aten::silu",        "aten::size",
-      "aten::sub",      "aten::add",         "aten::div",
-      "aten::mul",      "aten::eq",          "aten::lt",
-      "aten::ne",       "aten::__and__",     "aten::floor_divide",
-      "prim::Constant", "aten::expand_as",   "aten::repeat",
-      "aten::expand",   "aten::sum",         "aten::mean",
-      "aten::permute",  "aten::t",           "aten::transpose",
-      "aten::matmul",   "aten::bmm",         "aten::mm",
-      "aten::linear",   "aten::addmm"};
+      "aten::__and__",      "aten::add",      "aten::addmm",
```

Comment:
needs the `clang-format off` and `clang-format on` guard to avoid pre-commit formatting the code.

Reply:
It works!